networking/synthetic/stac/Dockerfile

```diff
@@ -47,31 +50,43 @@ RUN yum install --assumeyes \
 RUN wget ${stac_url}; \
     tar zxf STAC-N-Tools-Pkg20131015_jeder.tar.gz; \
```

Comment:
stac_url is a variable but the filename below that it tries to untar is hardcoded?